community-operators/jaeger/jaeger.v1.11.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,303 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: jaeger-operator.v1.11.0
+  replaces: jaeger-operator.v1.8.2
```

Comment:
This is the last version in the operator-hub (this repo) that is being replaced, although there was actually a version (1.10.0) in the Jaeger repo. I assume this should point to 1.8.2 rather than 1.10.0?

Reply:
This field should be moved to `spec.replaces` rather than in metadata

Reply:
And yes, this should point to the previous CSV in this repo. If you want users of version 1.10.0 on OLM to upgrade, I recommend including that CSV version in this PR.

Reply:
Moved.